### PR TITLE
Using underscore names in `setup.cfg`

### PR DESCRIPTION
**Pull request by aprotyas**
The previous names are now deprecated, so this commit migrates away from
them.

Signed-off-by: Abrar Rahman Protyasha <abrar@openrobotics.org>

### CODE REVIEW

**Review by sloretz — APPROVED** [review]

**Comment by artivis** [comment]
For context, since [`setuptools` v54.1.0](https://setuptools.pypa.io/en/latest/history.html#v54-1-0) the use of dashe in parameters is deprecated in favor of underscore.

I'll wait for #46 to be merged before merging this.

**Review by artivis — APPROVED** [review]

**Pull request merged**
